event/events.py

```diff
@@ -125,6 +127,10 @@ def character_gating_mod(self, events, name_event):
                     unlocked_slots.append(slot)
                     unlocked_slot_iterations.append(slot_iterations[slot])
 
+            # this means an impossible start has occured.
+            # i.e. no character can be retrieved given the starting char + check availability
+            assert len(unlocked_slots) > 0
```

Comment:
This will throw if you block out too many character slots or something along those lines. This is just a way to get context if you do something "illegal" with the new flags